index.bs

```diff
@@ -479,9 +497,9 @@ The following steps are taken:
 1. Let _entries_ be _registry_.[[RegistryData]].
 1. Let _pair_ be the entry in _entries_ such that _pair_.[[key]] is equal to _key_.
 1. If _pair_ does not exist, then throw a new *TypeError*.
-1. Let _stateValue_ be GetStateValue(_pair_.[[value]].[[State]]).
-1. Let _linkStateValue_ be GetStateValue("link").
-1. If _stateValue_ is less than _linkStateValue_, then throw a new *TypeError*.
```

Comment:
Question: I'm not sure about this condition. Why can we uninstall entries that are in `fetch`, `translate` or `instantiate`?